package/database/sqlinstance/composition.yaml

```diff
@@ -15,29 +15,41 @@ spec:
         apiVersion: rds.aws.upbound.io/v1beta1
         kind: SubnetGroup
         spec:
+          deletionPolicy: Delete
           forProvider:
-            region: us-west-2
+            # region: us-west-2 is default in the definition.yaml
             description: An excellent formation of subnetworks.
-          deletionPolicy: Delete
       patches:
+        # type: FromCompositeFieldPath is default
+        - fromFieldPath: spec.parameters.region
+          toFieldPath: spec.forProvider.region
+        # type: FromCompositeFieldPath is default
+        - fromFieldPath: spec.parameters.deletionPolicy
+          toFieldPath: spec.deletionPolicy
         - fromFieldPath: spec.parameters.clusterRef.id
           toFieldPath: spec.forProvider.subnetIdSelector.matchLabels[networks.aws.platformref.upbound.io/network-id]
     - name: RDSInstanceSmall
       base:
         apiVersion: rds.aws.upbound.io/v1beta1
         kind: Instance
         spec:
+          deletionPolicy: Delete
           forProvider:
-            region: us-west-2
+            # region: us-west-2
             dbSubnetGroupNameSelector:
               matchControllerRef: true
             instanceClass: db.t3.micro
             username: masteruser
             skipFinalSnapshot: true
             publiclyAccessible: false
             dbName: upbound # create custom database within the instance by default
-          deletionPolicy: Delete
       patches:
+        # type: FromCompositeFieldPath is default
```

Comment:
Can we use patch sets instead to make it consistent across the composition?

Reply:
Removed.

Reply:
Thanks for the inspiration. Using patchSets.